plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/model/CubridUniqueKey.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.ext.cubrid.model;
+
+import org.jkiss.dbeaver.ext.generic.model.GenericUniqueKey;
+import org.jkiss.dbeaver.model.struct.DBSEntityConstraintType;
+
+public class CubridUniqueKey extends GenericUniqueKey{
+
+	public CubridUniqueKey(CubridTable table, String name, String remarks, DBSEntityConstraintType constraintType,
+		boolean persisted) {
+		super(table, name, remarks, constraintType, persisted);
+	}

```

Comment:
Maybe we can use generic implementation directly?

Reply:
For `CubridUniqueKey`, it is same case as `CubridForeignKey`. Please kindly refer to my comment on `CubridTableForeignKey` above.

Reply:
I have fixed it by using generic implementation directly.